events/events.go

```diff
@@ -49,3 +49,9 @@ type LookupEvent struct {
 	Key      string
 	Duration time.Duration
 }
+
+// Ready is fired when the state of ringpop is changed to ready
```

Comment:
This comment is for public documentation, so can I suggest:

`Ready is fired when ringpop has successfully bootstrapped and is ready to receive requests and other method calls.`

Or something